common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -463,4 +463,15 @@ trait FeatureSwitches {
     exposeClientSide = true
   )
 
+  // Owner: Maria Livia Chiorean
+  val SharingComments = Switch(
+    SwitchGroup.Feature,
+    "sharing-comments",
+    "When ON, the user will be able to share comments",
+    owners = Seq(Owner.withGithub("marialivia16")),
+    safeState = Off,
+    sellByDate = never,
```

Comment:
@nicl @crifmulholland Any thoughts on how we can test this? One of the things we will be looking at is number of shares. We can track the number of clicks using the data-link-names.